en/SetupView/Sensors.md

```diff
@@ -1,42 +1,118 @@
+
+> **Note** Older ArduPilot firmware can be calibrated using the same process as PX4. 
+
+You need to rotate the vehicle randomly around all axes until the progress bar fills all the way to the right and the calibration completes. When the calibration completes you will get the following results:
+
+![ArduPilot Compass Calibration Onboard Result](../../images/setup/sensor_compass_ardupilot_onboard_calibration_result.jpg)
 
 This shows you the quality of the calibration for each compass. Using these values you can determine whether you may want to turn off usage of poorly performing compasses.
 
-### ArduPilot (older firmwares) and PX4
 
-For PX4 and older ArduPilot you will be guided through positioning the vehicle in a number of set orientations and rotating the vehicle through the request axis. To calibrate place the Vehicle in one of the incomplete orientations and hold it still, when asked to Rotate rotate the vehicle around the specific axis.
 
-![](../../images/setup/CompassImageCal.jpg)
+## Accelerometer
+
+To calibrate the flight controller accelerometers you will be asked to place and hold your vehicle a number of orientations (you will be prompted when to move between positions).
+
+The screens are different for PX4 and ArduPilot:
+
+* **PX4:** Position the vehicle as guided by the *images* on the screen. This is very similar to compass calibration.
+  
+  ![Accelerometer calibration](../../images/setup/sensor_accelerometer_positions_px4.jpg)
+  
+* **ArduPilot:** Position the vehicle based on text instructions in the center display. Click the **Next** button to capture each position.
+
 
 ## Level Horizon
-If after completing Accelerometer calibration you see that the horizon as shown in the HUD is not level you can calibrate the level horizon for your vehicle. You will be asked to place the vehicle in a level orientation while it captures the information.
 
-## CompassMot (ArduPilot only)
-CompassMot calibration is only recommended for vehicles that have only an internal compass and on vehicles where there is significant interference on the compass from the motors, power wires, etc. CompassMot only works well if you have a battery current monitor because the magnetic interference is linear with current drawn.
+If the horizon (as shown in the HUD) is not level after completing Accelerometer calibration you can calibrate the level horizon for your vehicle. You will be asked to place the vehicle in a level orientation while it captures the information.
+
+1. Place the vehicle in its level flight orientation on a level surface:
+
+  * For planes this is the position during level flight (planes tend to have their wings slightly pitched up!)
+  * For copters this is the hover position.
+
+1. Click the **Level Horizon** button.
+
+
+## Airspeed
+
+For Airspeed calibration you will need to keep your airspeed sensor out of any wind and then blow across the sensor. 
```

Comment:
This is from QGC. Is there anything else to this? (no airspeed sensor to test with). 

I am assuming you press OK to start the calibration. Get asked to blow across the sensor, then after a few seconds you get told calibration is complete. Correct?